lms/templates/courseware/course_about.html

```diff
@@ -112,8 +112,9 @@
         <div class="heading-group">
           <h1>
             ${course.display_name_with_default_escaped}
-            <button type="button">${course.display_org_with_default | h}</button>
           </h1>
+          <br />
```

Comment:
Nit: I don't like including break tags. Would it be possible to make the span into something that has some top margin?

Reply:
I was trying to avoid any styling changes, I would also normally never do this... but... :) sneaking in a quick fix mostly.